pages/about.html

```diff
@@ -4,9 +4,63 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../stylesheets/styles.css">
+  <title>Rose's About Me</title>
 </head>
 <body>
-  
+  <div class="wrapper about-me">
+    <nav class="nav-bar">        
+      <ul>
+        <li>
+          <h1 class="logo">RL</h1>
+        </li>
+        <li>
+          <a href="./index.html">Back to Main Page</a>
+        </li>
+        <li>
+          <a id="portfolio" href="portfolio.html">Portfolio</a>
+        </li>
+        <li>
+          <a id="github" href="https://github.com/rose-codes">GitHub</a>
+        </li>
+        <li>
+          <a id="linkedin" href="https://linkedin.com/in/rhle">LinkedIn</a>
+        </li>
+      </ul>
+    </nav>
+    <section class="about-me-hero">
+      <header class="about-me-header">
+        <div class="about-me-photo"></div>
+          <div class="about-me-photo photo"></div>
```

Comment:
Just for my own curiosity, why did you decide to display your photo as a background image instead of an `img` element?